README.md

```diff
@@ -36,7 +36,7 @@ You can achieve this in two ways:
 Then you are able to use the `SmartPaymentButtons` component where ever you want:
 
 ```javascript
-import SmartPaymentButtons from 'react-smart-payment-buttons';
+import { SmartPaymentButtons, SmartSubscriptionButtons } from 'react-smart-payment-buttons';
```

Comment:
I would prefer leaving the default export on `SmartPaymentButtons` so nobody runs into problems when upgrading the package. But I'd also add it as a named export to allow this kind of import, too.